docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -144,6 +152,13 @@ When using any database besides PostgreSQL, a JDBC driver jar file must be place
     iceberg.jdbc-catalog.connection-password=test
     iceberg.jdbc-catalog.default-warehouse-dir=s3://bucket
 
+.. _jdbc_limitations:
```

Comment:
had to specify these since the header names clashed and the anchor tags weren't randomly generated `#id1`.